docs/hub/hf-mcp-server.md

```diff
@@ -0,0 +1,61 @@
+# Hugging Face MCP Server
+
+The Hugging Face MCP (Model Context Protocol) Server connects your MCP‑compatible AI assistant (for example Codex, Cursor, VS Code extensions, Zed, or Claude Desktop) directly to the Hugging Face Hub. Once connected, your assistant can search and explore Hub resources and use community tools, all from within your editor or chat.
+
+## What you can do
+
+- Search and explore Hub resources: models, datasets, Spaces, and papers
+- Run community tools via MCP‑compatible Gradio apps hosted on [Spaces](https://hf.co/spaces)
+- Bring results back into your assistant with metadata, links, and context
+
+## Get started
+
+1. Open your MCP settings: visit https://huggingface.co/settings/mcp while logged in.
+
+2. Pick your client: select your MCP‑compatible client (for example Cursor, VS Code, Zed, Claude Desktop). The page shows client‑specific instructions and a ready‑to‑copy configuration snippet.
+
+3. Paste and restart: copy the snippet into your client’s MCP configuration, save, and restart/reload the client. You should see “Hugging Face” (or similar) listed as a connected MCP server in your client.
+
+<Tip>
+
+The settings page generates the exact configuration your client expects. Use it rather than writing config by hand.
+
+</Tip>
+
+![MCP Settings Example](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hf-mcp-settings.png)
+
+## Using the server
+
+After connecting, ask your assistant to use the Hugging Face tools. Example prompts:
+
+- “Search Hugging Face models for Qwen 3 Quantizatizations.”
+- “Find a Space that can transcribe audio files.”
+- “Show datasets about weather time‑series.”
+- “Create a 1024 x 1024 image of a dof anime style.”
```

Comment:
```suggestion
- “Create a 1024 x 1024 image of a dog anime style.”
```

I guess?

Reply:
it's supposed to be a cat (see the image below :p)

Reply:
Yeah, but the prompt is different, I thought it was a different example. If you want we can sync the prompt in the markdown to `create an image of a cat ghibli style 1024 x 1024`

Reply:
sure, done here: https://github.com/huggingface/hub-docs/pull/1938/commits/043a9d84fd01e3887b8b9192a32ffac9aa08912f